electron_app/src/spellcheck.js

```diff
@@ -0,0 +1,175 @@
+/* global require, process, _ */
+
+/* eslint-disable strict */
+const electron = require('electron');
+const osLocale = require('os-locale');
+const os = require('os');
+const semver = require('semver');
+const spellchecker = require('spellchecker');
+
+const { remote, webFrame } = electron;
+
+// `remote.require` since `Menu` is a main-process module.
+const buildEditorContextMenu = remote.require('electron-editor-context-menu');
+
+const EN_VARIANT = /^en/;
+
+// Prevent the spellchecker from showing contractions as errors.
+const ENGLISH_SKIP_WORDS = [
+  'ain',
```

Comment:
Indentation is wrong all over the shop, please review other code or see the style guide.